tests/diagnostics/test_climatenet.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+import pytest
+from earth2mip.diagnostic.climate_net import ClimateNet
+
+
+@pytest.mark.slow
+@pytest.mark.xfail
```

Comment:
why xfailed?

I propose you instantiate ClimatNet without loading the checkpoint and pass it some data at small resolution (small enough to make CPU test fast). This will give us some coverage of model code.

Reply:
The goal was to test the loading of the package rather than the model itself. I changed the name of the test. Not adding a manual instantiate test for the time being based on our discussions of moving things around / excepting coverage from some models.